iconservice/icon_service_engine.py

```diff
@@ -924,8 +924,7 @@ def _invoke_genesis(self,
 
         return tx_result
 
-    @classmethod
-    def _process_base_transaction(cls,
+    def _process_base_transaction(self,
```

Comment:
I am curious about why _process_base_transaction() is changed to an instance method.
Is it impossible to call cls._generate_logs_bloom() or any other reason?

Reply:
As you said, there are lots of ways of calling '_generate_logs_bloom', but I just thought that making _process_base_transaction as protected instance method and calling 'self._generate_logs_bloom()' is the proper way